project/unpackaged/myPackage/aura/TimesheetsIteration/TimesheetsIterationHelper.js

```diff
@@ -0,0 +1,14 @@
+({
+	initialList : function(cmp, event, helper) {
+        var action = cmp.get("c.getT");
+        action.setCallback(this, function(response) {
+ 		console.log(response.getReturnValue());
```

Comment:
remember to remove the "console.log" from the files before uploading into the repository.